configure.ac

```diff
@@ -61,6 +63,7 @@ AS_IF([test -n "$missing_gmock"],
 # Checks for libraries.
 AC_SEARCH_LIBS([__b64_ntop], [resolv])
 AC_SEARCH_LIBS([pthread_create], [pthread])
+AC_SEARCH_LIBS([evhtp_ssl_init], [evhtp])
```

Comment:
So it turns out you're not using this function, despite its name, might be something internal? If it's an internal function, that might disappear in the future, and it'd be generally better if it was something *we* used.

How about `evhtp_make_request`?